src/components/curriculum/CurriculumReactFundamentals.js

```diff
@@ -128,4 +129,33 @@ const CurriculumReactFundamentals = ({
   )
 }
 
+export const TargetAudienceList = () => (
```

Comment:
Why is this one an empty list?

Reply:
Because I plan to incrementally refactor all the curriculums to this. I'll continue after the holiday with it